### DISCUSSION TITLE
How to make sprite from multiple images?

### DISCUSSION THREAD

**Discussion by ghost** [kaplayjs/kaplay#76] [Q&A] [answered]
I have one image where I have Idle animation. And other different images for fall, hit, jump, double jump, etc.
How do I create a sprite from multiple images?
![image](https://github.com/marklovers/kaplay/assets/76583762/5b568d04-44e8-4bad-bbeb-e6330565bf78)

**Comment by lajbel** [comment, marked as answer]
You can do it passing an array to the `loadSprite` function 

```js
loadSprite("bean", ["/sprites/bean.png", "/sprites/ghosty.png"]);
```
